File: Modul 4/Soal 5/PRAK405-NIM-Nama.c
#include <stdio.h>

int main ()
{
    int i, j;
    int baris, pengali, total=0;

    scanf("%d %d", &baris, &pengali);

    for (i = 1; i <= baris; i++){
    
        int sum = 0;
        printf("(");

        for (j = 1; j <= i; j++){

            printf("%d * %d", j, pengali);
            sum += j * pengali;

            if (i>j){

                printf(") + (");
            }

            else if (j>i){

                printf(") + (");
            }   
        }
        printf(") = %d\n", sum);
        total += sum;
    }
    printf ("%d\n", total);
    return 0;
}

File: Modul 4/Soal 5/PRAK405-NIM-Nama.py
baris, pengali = map(int,input("").split())
total = 0

for i in range (1, baris + 1):
    sum_baris = 0
    print("(", end="")

    for j in range (1, i + 1):
        print(f"{j} * {pengali}", end="")
        sum_baris += j * pengali

        if j < i or i > j:
            print(") + (", end="")

    print(f") = {sum_baris}")
    total += sum_baris

print(f"{total}")